core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -106,12 +107,19 @@ public KafkaStore(SchemaRegistryConfig config,
     this.zkUtils = ZkUtils.apply(
         kafkaClusterZkUrl, zkSessionTimeoutMs, zkSessionTimeoutMs,
         JaasUtils.isZkSecurityEnabled());
-    this.kafkaTopicReader =
-        new KafkaStoreReaderThread<>(zkUtils, kafkaClusterZkUrl, topic, groupId,
-                                         Integer.MIN_VALUE, this.storeUpdateHandler,
-                                         serializer, this.localStore, this.noopKey);
     this.brokerSeq = zkUtils.getAllBrokersInCluster();
 
+    List<Broker> brokers = JavaConversions.seqAsJavaList(brokerSeq);
```

Comment:
I would have thought we'd rather populate the bootstrap brokers from configuration than from zookeeper.